include/rpp_backend.h.in

```diff
@@ -1,8 +1,7 @@
 #ifndef GUARD_RPP_BACKEND_H_IN
 #define GUARD_RPP_BACKEND_H_IN
 
-#cmakedefine01 GPU_SUPPORT
```

Comment:
why is the GPU_SUPPORT removed from here?

Reply:
The builds seemed to work without it